src/components/sbb-header/sbb-header.stories.js

```diff
@@ -23,15 +37,8 @@ const HeaderBasicTemplate = ({ children, ...args }) => [
       <sbb-menu-action icon-name="tick-small">English</sbb-menu-action>
     </sbb-menu>
   </sbb-header>,
-  <div>
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam sit amet malesuada augue. Morbi
-    eget tristique nisl, sit amet dapibus erat. Donec tempor, metus et aliquam ultrices, nulla mi
-    mollis urna, a lacinia mauris risus mattis massa. Quisque cursus sollicitudin enim in malesuada.
-    Maecenas nec hendrerit augue. Duis porttitor mattis molestie. Sed imperdiet velit at dui
-    ultrices, viverra scelerisque nisi dapibus. Nulla urna lectus, gravida eu dapibus vel, mattis
-    non turpis. Nunc interdum et justo sed faucibus. Vestibulum interdum commodo mi, sed eleifend
-    odio posuere in. Nunc non dui venenatis, eleifend est ut, varius odio. Quisque augue ante,
-    mollis eu lorem id, commodo cursus risus.
+  <div class="sbb-page-spacing" {...attributes}>
```

Comment:
in example expanded, the spacing should also be the expanded (sbb-page-spacing-expanded)